dev-libs/igraph/igraph-0.9.0.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+CMAKE_MAKEFILE_GENERATOR="emake"
```

Comment:
why is emake required?

Reply:
For the testing phase, as the default testing functions don't work.

Reply:
removed